src/main/scala/iliad/IliadPlugin.scala

```diff
@@ -24,6 +25,14 @@ object IliadPlugin extends AutoPlugin {
 
     /** Name of the generated app - required setting */
     val generatedAppName = commonKeys.generatedAppName
+
+    /** width in pixels of the view */
+    val width = commonKeys.width
+
+    /** height in pixels of the view */
+    val height = commonKeys.height
+
+    val targetOut = commonKeys.targetOut
```

Comment:
I don't think `targetOut` should be here.  The `autoImport` block is for any settings the plugin user should edit.  
`targetOut` is something we (plugin creators) set ourselves, and is not editable.